frontend/src/pages/TermsOfUse/__tests__/__snapshots__/termsOfUse.spec.tsx.snap

```diff
@@ -0,0 +1,120 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`matches snapshot 1`] = `
+<DocumentFragment>
```

Comment:
Could you create snapshot tests that render the entire ToU for different users (a regular user, and an admin user)? Since we have conditional logic to create these versions, it would be good to use the snapshot tests to create records of each ToU version in its entirety.